mephisto/abstractions/blueprint.py

```diff
@@ -227,10 +231,10 @@ def launch_assignment(
         Validate the assignment is prepared to launch, then run it
         """
         if assignment.db_id in self.running_assignments:
-            print(f"Assignment {assignment.db_id} is already running")
+            logger.debug(f"Assignment {assignment} is already running")
             return
 
-        print(f"Assignment {assignment.db_id} is launching with {agents}")
+        logger.debug(f"Assignment {assignment} is launching with {agents}")
```

Comment:
do you think it might be better to show `agent.db_id` rather than the object `agents` ?

Reply:
I've added repr functions for these - they now display as:
`Assignment Assignment(3165) is launching with [MockAgent(1034, in task), MockAgent(1035, in task)]`